coffee_cmd/src/cmd.rs

```diff
@@ -59,7 +61,11 @@ pub enum CoffeeCommand {
     Search { plugin: String },
     /// clean up remote repositories storage information
     #[clap(arg_required_else_help = false)]
-    Nurse {},
+    Nurse {
+        /// verify that coffee configuration is sane (without taking any action)
+        #[arg(short, long, action = clap::ArgAction::SetTrue)]
+        verify: bool,
```

Comment:
same for this

Reply:
if `verify` flag is provided, we will not take action. Otherwise, we fix the problems